config/rbac/role.yaml

```diff
@@ -27,7 +27,7 @@ rules:
   - patch
   - update
 - apiGroups:
-  - cluster.sigs.k8s.io
+  - cluster.x-k8s.io
```

Comment:
Should this be `bootstrap.cluster.x-k8s.io`?

Reply:
https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/pull/56/files/f94104b725fcd467b7ef495143db19abac482c58#diff-0406a3a4540c29330563040377417134R10

No, that is above. This is RBAC for the cluster objects.

Reply:
Which be scoped down in a future PR since we no longer need the watch on Machines